server/src/main/java/io/crate/exceptions/RoleUnknownException.java

```diff
@@ -21,11 +21,18 @@
 
 package io.crate.exceptions;
 
+import java.io.IOException;
 import java.util.Collections;
 import java.util.List;
 import java.util.Locale;
 
-public class RoleUnknownException extends RuntimeException implements ResourceUnknownException, UnscopedException {
+import org.elasticsearch.ElasticsearchException;
+import org.elasticsearch.common.io.stream.StreamInput;
+
+import io.crate.protocols.postgres.PGErrorStatus;
+import io.crate.rest.action.HttpErrorStatus;
+
+public class RoleUnknownException extends ElasticsearchException implements ResourceUnknownException, UnscopedException {
```

Comment:
Note that making it streamable is a bit of a side effect of having the `httpErrorStatus` and `pgErrorStatus` in `ElasticsearchException`. I'm not sure if this exception is currently ever raised on a non-handler node.

We could also introduce a `ResultException` interface for exceptions that are intended for serialization via pg/http to the end-user. Maybe even rename/repurpose `CrateException`?

Reply:
> Maybe even rename/repurpose CrateException?

Sounds good!

Reply:
I'll follow up on that with a separate PR